app/src/main/java/de/danoeh/antennapod/adapter/HorizontalFeedListAdapter.java

```diff
@@ -16,13 +16,23 @@
 import java.util.ArrayList;
 import java.util.List;
 
-public class HorizontalFeedListAdapter extends RecyclerView.Adapter<HorizontalFeedListAdapter.Holder> {
+import android.view.ContextMenu;
+import android.view.MenuInflater;
+import androidx.annotation.Nullable;
+
+public class HorizontalFeedListAdapter extends SelectableAdapter<HorizontalFeedListAdapter.Holder>
```

Comment:
`SelectableAdapter` is for multi-select, which we don't have here. Why is this parent class needed?